it/lessons/basics/basics.md

```diff
@@ -112,6 +146,9 @@ iex> "foo\nbar"
 "foo\nbar"
 ```
 
+Oltre a questi tipi di base, Elixir supporta tipi di dati piú compliessi.
+Impareremo di piú su questi durante le lezioni sulle [collezioni](../collections/) and [funzioni](../functions/).
```

Comment:
There's an `and` in the italian phrase. It should be `e`.

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b